core/ecschema-metadata/src/UnitConversion/UnitConversion.ts

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Constant, Unit } from "../ecschema-metadata";
+import { SchemaItemType } from "../ECObjects";
+import { Float } from "./Float";
+
+export class UnitConversion {
+  /** @internal */
+  constructor(
+    public readonly factor: number = 1.0,
+    public readonly offset: number = 0.0
+  ) {}
+
+  /** @alpha */
+  public evaluate(x: number): number {
+    return this.factor * x + this.offset;
+  }
+
+  /** @internal */
+  public inverse(): UnitConversion {
+    // y = mx + c
+    // => x = (y-c)/m
+    const inverseSlope = 1.0 / this.factor;
+    return new UnitConversion(inverseSlope, -this.offset * inverseSlope);
+  }
+
+  /** @internal */
+  public compose(map: UnitConversion): UnitConversion {
```

Comment:
'map' no longer makes sense as a parameter name.  Also remove comments they don't add much but bulk to the code